### ISSUE TITLE
Remove 'All rights reserved' from the copyright headers

### ISSUE THREAD

**Issue by vmoroz** [microsoft/react-native-windows#3933]
The links below may require Microsoft credentials: https://docs.opensource.microsoft.com/changelog.html

**Summary from the link:**
July 2019
The term All Rights Reserved is no longer recommended to be in MIT license copyright headers.

The new headers should be:
```
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.
```
See: https://docs.opensource.microsoft.com/content/releasing/copyright-headers.html

**Comment by NickGerleman** [comment]
@kmelmon why did you close this one?

**Comment by kmelmon** [comment]
> @kmelmon why did you close this one?

because I thought you had addressed it.  Maybe I mis-read the PR comment: "This is a quick regex replace. Addresses #3933 to remove "All rights reserved" and #3797 to replace the Facebook license banner." ?

**Comment by NickGerleman** [comment]
@kmelmon the PR was closed instead of merged, There were some concerns about replacing Facebook license headers that I think we need to follow up on so I abandoned it for now,

**Comment by kmelmon** [comment]
Sorry about that!  I reopened #3797 as well.

**Comment by NickGerleman** [comment]
ACoates fixed this with #5083
